lib/logasm/adapters/logstash_adapter.coffee

```diff
@@ -0,0 +1,31 @@
+winston = require('winston')
+_  = require 'underscore'
+LogstashUDP = require('winston-logstash-udp').LogstashUDP
+
+class LogstashAdapter
+  constructor:(level, service, {host, port}) ->
+    @logger = new winston.Logger
+
+    options =
+      port: port or 5229
+      appName: service if service
+      host: host or "127.0.0.1"
+      level: level or 'info'
+      timestamp: winston.timestamp
+
+    @logger.add LogstashUDP, options
+
+  log:(level, args) ->
+    args = _.extend({}, args)
+    message = args['message']
+
+    if message
+      delete args['message']
+      if message instanceof Object
+        @logger.log level, "#{message.stack}", args
```

Comment:
at least in the stdout adapter. not sure how we should log it here. maybe add the stack as a metadata